src/Setup/WixBA/NoSnapBehavior.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
+// This code was posted by bj0 at: http://stackoverflow.com/questions/2470685/how-do-you-disable-aero-snap-in-an-application
+// Only used portions of this sample code needed to address WIXBUG #5262 (The DragMoveBehavior code was throwing an exception).
+
+using System;
+using System.Windows;
+using System.Windows.Interactivity;
+
+namespace WixToolset.UX
+{
+    /// <summary>
+    /// behavior that makes a window/dialog not resizable while clicked.  this prevents
+    /// the window from being snapped to the edge of the screen (AeroSnap).
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public class NoSnapBehavior<T> : Behavior<T> where T : FrameworkElement
```

Comment:
I thought we discussed that it should be resizable, and that the maximize/restore button would be available.  In that case, we shouldn't be fighting the Aero Snap.  Though this probably should be put in v3.